Updated the release process for TypeScript packages (#117)
ckeditor/ckeditor5-package-generator: packages/ckeditor5-package-tools/README.md

* `dll:build` &ndash; prepares a file compatible with [CKEditor 5 DLL](https://ckeditor.com/docs/ckeditor5/latest/builds/guides/development/dll-builds.html) that exposes plugins from the package,
* `translations:collect` &ndash; collects translations context and prepares them to be sent to [Transifex](https://www.transifex.com/),
* `translations:upload` &ndash; uploads collected contexts to [Transifex](https://www.transifex.com/),
* `translations:download` &ndash; downloads translated contexts from [Transifex](https://www.transifex.com/),
* `export-package-as-javascript` &ndash; changes `main` entry in `package.json` file to point to a `.js` file,
* `export-package-as-typescript` &ndash; changes `main` entry in `package.json` file to point to a `.ts` file.

There are two ways to integrate these scripts, either with [npm scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts) or Node.js scripts.

### Integration with npm scripts

> Available scripts can be called via npm scripts in `package.json` file, eg:

Reviewer: ```suggestion
Available scripts can be called via npm scripts in the `package.json` file, e.g.:
```

```json
{
  "dll:build": "ckeditor5-package-tools dll:build"
}
```

> In case of `translations:download` and `translations:upload`, additional argument is required:

Reviewer: ```suggestion
The `translations:download` and `translations:upload` tasks require the `--transifex` modifier:
```

```json
{
  "translations:download": "ckeditor5-package-tools translations:download --transifex [API URL]"
}
```

### Integration with Node.js scripts

> Available scripts can be called manually in node scripts, eg:

Reviewer: ```suggestion
Available scripts can be called manually as Node scripts, e.g.:
```
